src/components/charts/PercentageStackedBarChart.jsx

```diff
@@ -0,0 +1,76 @@
+import { BarChart, Bar, Legend, Tooltip, XAxis, YAxis, CartesianGrid,
+} from 'recharts';
+import PropTypes from 'prop-types';
+import React from 'react';
+import _ from 'underscore';
+import { localTheme } from '../../localconf';
```

Comment:
we expect to not see any localtheme or localconf in the component at all.
The component should be self-contained and everything will be loaded as properties